lib/models/situation.js

```diff
@@ -140,6 +140,7 @@ var IndividuDef = {
     lienParente: { type: String, enum: ['fils', 'neveu', 'aucun', 'autre'] },
     role: { type: String, enum: ['demandeur', 'conjoint', 'enfant'] },
     ressources: [RessourceSchema],
+    ressourcesApproxAnnuel: Object,
```

Comment:
To preserve consistency  :monkey_face:  ?
Would approxRessourcesAnnuelles be better ?